app/Knights/Preparation.py

```diff
@@ -0,0 +1,29 @@
+class PrepareKnight:
```

Comment:
Rename this class to Knight because it's not proper way to name class using verb. And also rename package Knights and modules BATTLE, KNIGHTS, Preparation to lowercase